### PR TITLE
Update etcd upgrade notice for 1.6 release

### PR DESCRIPTION
**Pull request by calebamiles**
**What this PR does / why we need it**:

- Specifies flag to set on API Server
- Fixes link to warning in `CHANGELOG`

**Special notes for your reviewer**:

please cherry pick to 1.6 branch

**Release note**:

 `NONE`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -576,7 +576,14 @@ backup.
...
 Also, please note:
 - using `application/vnd.kubernetes.protobuf` as the media storage type for 1.6 is default but not required
-- the ability to rollback to etcd2 can be preserved by continuing to use `application/json` as the storage media type. This can be changed to `application/vnd.kubernetes.protobuf` at a later time.
+- the ability to rollback to etcd2 can be preserved by setting the storage media type flag on `kube-apiserver`
+
+  ```
+  --storage-media-type application/json
+  ```
+  
+  to continue to use `application/json` as the storage media type which changed to `application/vnd.kubernetes.protobuf`
```
Comment (bgrant0607) [inline CHANGELOG.md]: can be changed to

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,7 +20,7 @@
     - [Volume](#volume)
   - [Notable Features](#notable-features)
     - [Autoscaling](#autoscaling)
-    - [DaemonSets](#daemonsets)
+    - [DaemonSet](#daemonset)
```
Comment (enisoc) [inline CHANGELOG.md]: I think you forgot to change the actual heading in Notable Features.

**Comment by bgrant0607** [comment]
@k8s-bot non-cri node e2e test this

**Comment by bgrant0607** [comment]
Merging to fix 1.6 release notes.

**Pull request merged**